py/server/tests/test_udf_return_java_values.py

```diff
@@ -210,7 +212,7 @@ def f() -> np.ndarray[np.int64]:
 
         with self.assertRaises(DHError) as cm:
             t = empty_table(10).update(["X1 = f()"])
-        self.assertIn("not support multi-dimensional arrays", str(cm.exception))
+        self.assertIn("java.lang.ClassCastException: class [[J cannot be cast to class [J", str(cm.exception))
```

Comment:
Can we get a more python-user-friendly error message, or is the whole message python friendly, and this is just a snippet?

Reply:
It is the most important part of the message (including the full trackback etc.) and it is emitted out of the engine. I don't know there is a safe and easy way to make it more Python-user-friendly.